modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/visualization_modes/table/Table.tsx

```diff
@@ -125,7 +125,7 @@ const EditFDSFieldModalContent = ({
 	);
 
 	const [fdsFieldSortable, setFSDFieldSortable] = useState<boolean>(
-		fdsField.sortable ?? fdsField.type !== EFieldType.OBJECT
+		fdsField.sortable
```

Comment:
Naming here left me confused for a second, thinking why do we need this state, isn't this the same? But it's not, it's just misleadingly named. I think `fdsFieldSortable` should really be `fieldSortable` or just `sortable`. The broader idea in DSM naming convention is that `fds*` prefix implies that this represents stored entity, later on we could rename `fdsField` to `dsmTableColumn`. On the other hand, just `field*` prefix should contain frontend-only specifics and OpenAPI specifics, where it should remain as it is.